### ISSUE TITLE
startStream not working in web

### ISSUE THREAD

**Issue by naveenbharadwaj19** [llfbandit/record#324]
**Package version**
5.1.0

**Environment**
 - OS: windows
 - Browser : Chrome,edge

I've tried to listen to the stream once I start recording, but the problem is that I'm not receiving any data in the stream, and the print statement isn't getting invoked. I've tested the same app on Android with` AudioEncoder.pcm16bits`, and it's working fine.

```dart

class AudioRecorderWidget extends StatefulWidget {
  @override
  _AudioRecorderWidgetState createState() => _AudioRecorderWidgetState();
}

class _AudioRecorderWidgetState extends State<AudioRecorderWidget> {
  bool _isRecording = false;
  AudioRecorder recorder = AudioRecorder();

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Audio Recorder'),
      ),
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            ElevatedButton(
              onPressed: _isRecording ? null : _startRecording,
              child: Text('Start Recording'),
            ),
            SizedBox(height: 20),
            ElevatedButton(
              onPressed: _isRecording ? _stopRecording : null,
              child: Text('Stop Recording'),
            ),
          ],
        ),
      ),
    );
  }

  void _startRecording() async {
    bool isAvailable = await recorder.hasPermission();
    if (isAvailable) {
      setState(() {
        _isRecording = true;
      });
      var stream = await recorder
          .startStream(const RecordConfig(encoder: AudioEncoder.pcm16bits,
          ));

      stream.listen((event) {
        print("listening....");
      });
    } else {
      // Handle permissions not granted
      print('Permissions not granted');
    }
  }

  void _stopRecording() async {
    setState(() {
      _isRecording = false;
    });
    await recorder.stop();
  }
}
```

**Comment by llfbandit** [comment]
Does your browser logs something in the console? No exception in your `_startRecording` method?
Which version of `record_web` do you use? Current version is `1.1.1`

I can't reproduce it.

**Comment by naveenbharadwaj19** [comment]
I'm not getting any logs. I've added try and catch no exception found. Where to find record_web version? i've added only `record` package

```dart

 void _startRecording() async {
    try {
      bool isAvailable = await recorder.hasPermission();
    if (isAvailable) {
      setState(() {
        _isRecording = true;
      });
      var stream = await recorder
          .startStream(const RecordConfig(encoder: AudioEncoder.pcm16bits,
          ));

      stream.listen((event) {
        print("listening");
      });
    } else {
      // Handle permissions not granted
      print('Permissions not granted');
    }
    } catch (e) {
      print("error in start recording : $e");
    }
    
  }

**Comment by llfbandit** [comment]
You can find `record_web` reference in your `pubspec.lock` file.
Do you also reproduce it with the example project?

I will need more info otherwise I won't be able to help you here...

**Comment by naveenbharadwaj19** [comment]
record_web:
    dependency: transitive
    description:
      name: record_web
      sha256: "0ef370d1e6553ad33c39dd03103b374e7861f3518b0533e64c94d73f988a5ffa"
      url: "https://pub.dev"
    source: hosted
    version: "1.1.0"
    
  No i haven't tried with example project

**Comment by DanieleMarchei** [comment]
I had the same problem with my code and the example project.
The issue was that my Flutter SDK version was too old, so I ran
```bash
flutter upgrade --force
```
and then
```bash
flutter pub get
```
After that, the example project and my code started working on the browser too.

**Comment by llfbandit** [comment]
This issue should be resolved with 1.1.1 version.
